### PR TITLE
Restore :current pseudo class

### PR DESCRIPTION
**Pull request by estelle**
This PR is in DRAFT mode until we decide whether we should add current back in or not.

### CODE REVIEW

File: files/en-us/web/api/console/timelog/index.md [hunk 1]
```diff
@@ -35,17 +35,7 @@ timeLog(label)
...
-```
-default: 1042ms
-```
-
-If an existing `label` is included:
-
-```
-timer name: 1242ms
-```
+None ({{jsxref("undefined")}}).
```
Comment (estelle) [inline files/en-us/web/api/console/timelog/index.md:38]:
```suggestion
If no label parameter included:

```
default: 1042ms
```

If an existing `label` is included:

```
timer name: 1242ms
```
```

File: files/en-us/web/css/_colon_current/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: ':current'
+slug: Web/CSS/:current
+page-type: css-pseudo-class
+browser-compat: css.selectors.current
+---
+{{CSSRef}}
+
+The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents the element, or an ancestor of the element, that is currently being displayed. For example in a video with captions which are being displayed by [WebVTT](/en-US/docs/Web/API/WebVTT_API).
```
Comment (jpmedley) [inline files/en-us/web/css/_colon_current/index.md:9]:
```suggestion
The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents the element or an ancestor of the element, that is currently being displayed. For example in a video with captions which are being displayed by [WebVTT](/en-US/docs/Web/API/WebVTT_API).
```

Comment (jpmedley) [reply files/en-us/web/css/_colon_current/index.md:9]: The phrase 'element or an ancestor of the element' are elements in a series. Elements in a series only require commas when there are three or more. Also, 'or an ancestor of the element' is restrictive and therefore should have no comma.

(I had to dig out the grammar book on this one. Let me know if I've overlooked anything.)

**Review by jpmedley — DISMISSED** [review]

File: files/en-us/web/css/_colon_current/index.md (hunk 2 above)
Comment (dipikabh) [reply files/en-us/web/css/_colon_current/index.md:9]: That is the correct rule for comma with conjunctions! 🙂

Comment (dipikabh) [inline files/en-us/web/css/_colon_current/index.md:9]:
```suggestion
The **`:current`** [CSS](/en-US/docs/Web/CSS) [pseudo-class](/en-US/docs/Web/CSS/Pseudo-classes) selector is a time-dimensional pseudo-class that represents an element or the ancestor of an element that is currently being displayed. For example, this pseudo-class can be used to represent a video that is being displayed with captions by [WebVTT](/en-US/docs/Web/API/WebVTT_API).
```
also dropping the comma before 'that'

**Review by dipikabh — CHANGES_REQUESTED** [review]
A few more changes needed in addition to @jpmedley's

**Review by dipikabh — APPROVED** [review]
LGTM

**Pull request merged**